### DISCUSSION TITLE
Difference between Immich server and Immich Web UI

### DISCUSSION THREAD

**Discussion by cgeheim** [immich-app/immich#8882] [Q&A]
Is there a Difference between Immich server and Immich Web UI.

Can. Somebody explane this?

I use in Docker only the server version. is it better to use the Web UI ?

I am working with Windows 11

**Comment by alextran1502** [comment]
Hello, the web needs the server to operate. The server has all the logics regarding photos and videos management while the web serves as the client to view and interact with them
